cms/static/js/programs/routers/core_app_program_router.js

```diff
@@ -0,0 +1,67 @@
+define([
```

Comment:
@AlasdairSwan may have a suggestion more in-line with our conventions in this space. At the very least, I think the name of the file can change.